examples/nil-lifting/nil_lifting.bal

```diff
@@ -1,47 +1,58 @@
 import ballerina/io;
 
-type address record {
+type Address record {
     string line01;
     string line02;
     string city;
     string state;
     string zipcode;
 };
 
-// According to this, `addr` and `guardian` fields may or may not contain a value. And either all the fields must be
-// assigned with default values e.g. `string name = ""` or be initialized before use.
-type person record {
+// According to the `Person` record type descriptor, the `addr` field could either hold an `Address` record or the value
+// `nil` (`()`).
```

Comment:
Should we be code-formatting the word nil, seeing as it's not a keyword nor the actual representation of nil used in code?